src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -893,19 +893,19 @@ to_Port(ParsedPort const& parsed, std::ostream& log)
 
     if (! parsed.ip)
     {
-        log << "Missing 'ip' in [" << p.name << "]\n";
+        log << "Missing 'ip' in [" << p.name << "]";
         Throw<std::exception> ();
```

Comment:
Food for thought: why not do something like:

Throw<std::runtime_exception>("Missing 'ip' in [" + p.name + "]");

Reply:
@miguelportilla suggested a similar change.

I had gone down that route (along with eliminating the exceptions altogether), but it was starting to touch more code than I was comfortable with for this targeted change. I wasn't sure if some callers wanted to distinguish logging from exception handling.